### DISCUSSION TITLE
encode/httpx help menu error

### DISCUSSION THREAD

**Discussion by Teicu** [projectdiscovery/httpx#516] [Q&A] [answered]
Hi all 

option -l and -h not works anymore 

So which is the command to run this tool with a hosts file? 

```
httpx -list /targets.txt -silent -probe 
Usage: httpx [OPTIONS] URL

Error: no such option: -l
``` 
```

admin@Pentest ~ % httpx -h 
Error: -h option requires 2 arguments
```
Thank you

**Comment by ehsandeep** [comment, marked as answer]
@Teicu you must be running something else, probably https://github.com/encode/httpx,  this is not an output of httpx cli option.

**Reply by spector012** [reply]
@ehsandeep, pls how do i run the right httpx?

**Comment by forgedhallpass** [comment]
@Teicu  @spector012 
Please refer to: https://github.com/projectdiscovery/httpx/discussions/775
